pkg/controller/pxcbackup/controller.go

```diff
@@ -99,6 +100,8 @@ type ReconcilePerconaXtraDBClusterBackup struct {
 	clientcmd           *clientcmd.Client
 	chLimit             chan struct{}
 	bcpDeleteInProgress *sync.Map
+
+	newStorageClientFunc storage.NewClientFunc
```

Comment:
I don't get the point of having this in the reconciler struct. I thought it's for tests but I don't see it's used there.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1600/commits/03c867973553d3ff0f3da368fd1ba864f9c38748